From 628c1aeb34ddf0c1b6bac9f2a07891cfbf83e75e Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Thu, 8 Feb 2018 11:32:15 +0800
Subject: [PATCH] arm64: dts: rockchip: enable cdn DP for rk3399 sapphire

As we had posted the DP firmeware to /lib/firmware/rockchip,
so you need check this path in your rootfs first, if enable the DP.

Change-Id: Id24a42edc9d6cd0b0de0a2dfefe2727df3965b67
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
index cfd08f030dab..8e4b9e6141b5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
@@ -192,6 +192,12 @@
 	};
 };
 
+&cdn_dp {
+	status = "okay";
+	extcon = <&fusb0>;
+	phys = <&tcphy0_dp>;
+};
+
 &cpu_l0 {
 	cpu-supply = <&vdd_cpu_l>;
 };
-- 
2.35.3

